plugins/discovery-file/build.gradle

```diff
@@ -27,3 +31,36 @@ bundlePlugin {
         into 'config'
     }
 }
+
+task seedTask << { }
+// setup the initial cluster with one node that will serve as the seed node
+// for unicast discovery
+ClusterConfiguration config = new ClusterConfiguration(project)
+config.distribution = 'zip'
+config.clusterName = 'discovery-file-test-cluster'
+List<NodeInfo> nodes = ClusterFormationTasks.setup(project, seedTask, config)
+
+// write the unicast_hosts.txt file to a temporary location to be used by the second cluster
+task setupSeedNodeAndUnicastHostsFile(type: DefaultTask) {
+    mustRunAfter(precommit)
+    dependsOn(seedTask)
+    doLast {
+        // write the unicast_hosts.txt file to a temp file in the build directory
+        File unicastHostsFile = file('build/unicast_hosts.txt')
```

Comment:
I would set this path up outside of this task, so you don't need to copy it below for the extraConfigFile line. Also, can you put this under cluster subdir somewhere? That is where the rest of the integ test cluster stuff is (rather than a random file at the root of `build`.